docs/cody/capabilities/ignore-context.mdx

```diff
@@ -1,10 +1,151 @@
 # Cody Ignore
 
-<Callout type="note">This feature is currently in the Experimental stage and only supported with VS Code extension.</Callout>
+## Enterprise
 
-This guide will walk you through the process of ignoring specific files or folders in your codebase when using Cody. 
+<Callout type="note">
+Client Support for Cody Context Filters
```

Comment:
```suggestion
Enterprise instances admins can use the Cody Context Filter to determine which repositories Cody is allowed to use as context in its requests to third-party LLMs. This functionality is currently experimental and is only supported under the following conditions:
```

Reply:
We later communicate about clients' version restrictions. If clients meet them, Cody Ignore is probably not experimental yet 🤔